### ISSUE TITLE
Shebang in mqtt package breaks webpack

### ISSUE THREAD

**Issue by chasegiunta** [pushy-me/pushy-electron#1]
```
ERROR  Failed to compile with 1 errors                                                                                                                                 9:56:37 PM

 error  in ./node_modules/mqtt/mqtt.js

Module parse failed: Unexpected character '#' (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> #!/usr/bin/env node
| 'use strict'
| 

 @ ./node_modules/pushy-electron/util/mqtt.js 1:13-28
 @ ./node_modules/pushy-electron/lib/Pushy.js
 @ ./src/background.js
 @ multi ./src/background.js

 ERROR  Build failed with errors.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Changing `const mqtt = require('mqtt');` to `const mqtt = require('mqtt/dist/mqtt.js');` fixes the issue.

**Comment by chasegiunta** [comment]
Actually, looks like this may be fixed in an upcoming release...

https://github.com/mqttjs/MQTT.js/commit/e3ea11a2f941ea1a5c98f3944311e41ac7e30da2

**Comment by pushy-me** [comment]
Hi @chasegiunta,
Thanks so much for reporting this issue.

We've applied a fix in https://github.com/pushy-me/pushy-electron/commit/7cfcab6903dc628581201c6060c71950f15d2d18 and published `1.0.4`. 🎉

**Comment by chasegiunta** [comment]
Appreciate it!
